### PR TITLE
Make the Operator works

### PR DESCRIPTION
**Pull request by vincent-pli**
- Correct git.apache.org/thrift.git in go.mod
- Add resources copy in Dockerfiles
- Correct role binding

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -35,6 +33,7 @@ github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc/go.mod h1:LOuy
 github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf h1:qet1QNfXsQxTZqLG4oE62mJzwPIB8+Tee4RNCL9ulrY=
 github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf/go.mod h1:ybxpYRFXyAe+OPACYpWeL0wqObRcbAqCMya13uyzqw0=
 github.com/anmitsu/go-shlex v0.0.0-20161002113705-648efa622239/go.mod h1:2FmKhYUyUczH0OGQWaF5ceTx0UBShxjsH6f8oGKYe2c=
+github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999/go.mod h1:cp2SuWMxlEZw2r+iP2GNCdIi4C1qmUzdZFSVb+bacwQ=
```
Comment (sthaha) [inline go.sum]: @vincent-pli why is this change need?

File: go.mod [hunk 2]
```diff
@@ -25,6 +25,7 @@ replace (
 )
 
 replace (
+	git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```
Comment (sthaha) [inline go.mod]: why do we need this? aren't they both the same and allows for thrift.git to be be moved anywhere?

File: build/Dockerfile [hunk 3]
```diff
@@ -10,6 +10,8 @@ COPY build/_output/bin/operator ${OPERATOR}
 COPY build/bin /usr/local/bin
 RUN  /usr/local/bin/user_setup
 
+COPY deploy/resources /deploy/resources
```
Comment (sthaha) [inline build/Dockerfile]: +1 , thank you!

File: go.mod (hunk 2 above)
Comment (vincent-pli) [reply go.mod]: I don't know, I hit the dependency resolve issue when run go mod, and then find this:
https://github.com/jenkins-x/jx/issues/3320
it resolve my issue.

**Pull request merged**